### ISSUE TITLE
Uncaught TypeError: Found non-callable @@iterator

### ISSUE THREAD

**Issue by metaperl** [pyscript/pyscript#528]
### Checklist

- [X] I added a descriptive title
- [X] I searched for other issues and couldn't find a solution or duplication
- [X] I already searched in Google and didn't find any good information or help

### What happened?

I attempted to load [the simple clock demo](https://pyscript.net/examples/simple_clock.html) and after a few minutes I started looking in the developer tools from Google Chrome and found this error:

```
pyconfig.ts:106 Uncaught TypeError: Found non-callable @@iterator
    at PyConfig.connectedCallback (pyconfig.ts:106:34)
    at main.ts:22:34
    at main.ts:35:2
```

### What browsers are you seeing the problem on? (if applicable)

Chrome

### Console info

```shell
pyconfig.ts:106 Uncaught TypeError: Found non-callable @@iterator
    at PyConfig.connectedCallback (pyconfig.ts:106:34)
    at main.ts:22:34
    at main.ts:35:2
```

### Additional Context

I am using Windows 10, Google Chrome Version 102.0.5005.115 (Official Build) (64-bit)

**Comment by marimeireles** [comment]
Can reproduce, working on it.
Thanks for the issue!

**Comment by marimeireles** [comment]
See discussion here: https://github.com/pyscript/pyscript/pull/523
I'm not sure what's happening but it seems like some kind of intermittent issue.
I'm not going to close this issue because I'm not sure if it's fixed. Or if it's under our control at all.
I just don't understand what's happening 😅

**Comment by mitsuo0114** [comment]
Hi, I've got same error when I misconfigure `py-config`.

See Correct case [1], and Wrong case [2]
Both yaml in `py-config` should be correct YAML format. But these cause structure difference ([3], [4]). 

and the example on `get-started.md` is showing wrong one.
https://github.com/pyscript/pyscript/blob/main/docs/tutorials/getting-started.md#the-py-config-tag

I'm not familiar with how appConfig.write (writable object) works, but I'm afraid these may happen in other cases.
https://svelte.dev/docs#run-time-svelte-store-writable

[1] Correct Case : 
```html
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <py-config>
- autoclose_loader : false
    </py-config>
    <script defer src="https://pyscript.net/alpha/pyscript.js"></script>
</head>
<body></body>
</html>
```

[2] Wrong Case:
```html
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <py-config>
autoclose_loader : false
    </py-config>
    <script defer src="https://pyscript.net/alpha/pyscript.js"></script>
</head>
<body></body>
</html>
```

[3] Correct Case
```json
[
  {
    "autoclose_loader": false
  }
]
```

[4] Wrong Case
```json
{
  "autoclose_loader": false
}
```

https://yaml-online-parser.appspot.com/

**Comment by ymd-h** [comment]
I encountered the same issue.

If we look generated CDN script, the error happens at `Object.assign({}, ...loadedValues)`.
If we obeys the document, `js-yaml.load` returns `Object` (not `Array`), spread operator becomes error.

https://pyscript.net/alpha/pyscript.js

```javascript
    class PyConfig extends BaseEvalElement {
        constructor() {
            super();
        }
        connectedCallback() {
            this.code = this.innerHTML;
            this.innerHTML = '';
            const loadedValues = load(this.code);
            if (loadedValues === undefined) {
                this.values = {
                    autoclose_loader: true,
                };
            }
            else {
                this.values = Object.assign({}, ...loadedValues);
            }
```

Semantically, `<py-config>` includes only a single config entry, so that I think it is better to keep config syntax at the document unchanged and to fix the `Object.assign()` sentence.

I am not familiar with TypeScript, so that I don't know why TypeScript adds spread operator for this simple assignment.
(The reason might be missing type of `loadedValues`?)

https://github.com/pyscript/pyscript/blob/eb31978488b465513c1b91b54ba51a821697f32c/pyscriptjs/src/components/pyconfig.ts#L135-L163

**Comment by mitsuo0114** [comment]
Thanks to @ymd-h 's post, I identify these parts are fixed recently in PR[1] and fixed on unstable version [2]
According to "last-modified" HTTP response header in /alpha/pyscript.js, it was released Thu, 19 May 2022 17:58:53 GMT.

[1]
https://github.com/pyscript/pyscript/pull/457/files#diff-564ff679a21992ceb7761245d07c014eba65f5ae15beaffe9ac0b7da771f987a

https://pyscript.net/unstable/pyscript.js

**Comment by ymd-h** [comment]
@mitsuo0114 
Thank you for your quick reply!

I also confirmed the unstable version worked well.

**Comment by JeffersGlass** [comment]
This issue is confirmed to be fixed in the current unstable version (2022.09.1-RC2).
